6/dotnetcore-3.1.102/Dockerfile

```diff
@@ -0,0 +1,53 @@
+ARG detect_latest_release_version=6.1.0
+
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1.102
+
+MAINTAINER Forest Keepers
+
+ARG detect_latest_release_version
+ENV DETECT_LATEST_RELEASE_VERSION $detect_latest_release_version
+ENV DETECT_VERSION_KEY $detect_latest_release_version
+ENV DOTNET_RUNTIME_VERSION_2 2.1.16
+
+# Install utilities
```

Comment:
The following can be extracted into a one-liner. I.e: RUN `apt-get update -y && apt-get install -y wget unzip`

Reply:
I've created a suggested code change for this.